files/init.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash 
 
 set -e
-sed -i "s/HOSTNAME/$C9_HOSTNAME/g" /home/ubuntu/workspace/README.md
\ No newline at end of file
+
+cd
+./replica_set.sh
```

Comment:
I'm guessing that Cloud 9 runs this file at some point, probably when creating a workspace.  We could also not put this here, and tell students in the README how to run the exercise.